OracleCollection.js

```diff
@@ -1229,11 +1253,15 @@ export default class OracleCollection {
     return this.getCollectionConnection()
       .then(conn => {
         localConn = conn;
-        return this._oracleCollection.find().remove();
+        if (typeof DB_VERSION === 'undefined' || DB_VERSION === '23') {
```

Comment:
Shouldn't this be 
`if (typeof DB_VERSION !== 'undefined' && DB_VERSION !== '23')`

Reply:
Never mind, I read it wrong...